Tangem/Common/Appearance/Fonts.swift

```diff
@@ -11,71 +11,254 @@ import SwiftUI
 enum Fonts {
     enum Regular {
         /// weight: regular, size: 34
-        static let largeTitle: Font = .system(size: 34, weight: .regular)
+        static var largeTitle: Font {
+            if FeatureProvider.isAvailable(.dynamicFonts) {
+                return DynamicFonts.largeTitle
+            } else {
+                return StaticFonts.largeTitle
+            }
+        }
```

Comment:
@tureck1y типа такого

```suggestion
        static let largeTitle: Font = FeatureProvider.isAvailable(.dynamicFonts) ? DynamicFonts.largeTitle : StaticFonts.largeTitle
```

Reply:
ага, обновил